src/app/model/User.ts

```diff
@@ -72,10 +72,12 @@ class User extends BaseModel {
       id: missionId,
       ...mission,
     });
+    mission.id = missionId;
 
     //save mission in firestore
     try {
-      await collection.doc(missionId).set(sanitizedMission);
+      await collection.doc(missionId).set(mission);
+      //await collection.doc(missionId).set(sanitizedMission);
```

Comment:
We were using this.load() to merge default data with the passed in data. If we do not use this, we don't create missions with default data.

Reply:
@azmainamin , my bad!  I didn't mean for that to get checked in ... I was messing around and left that in by mistake.  Good catch.